community/athens-guide/feature-list/graph-visualization.md

```diff
@@ -1,2 +1,22 @@
-# Graph Visualization
+# Graph Visualization 
 
+<img width="1440" alt="Screen Shot 2021-06-05 at 4 48 29 PM" src="https://user-images.githubusercontent.com/80150109/120890130-7b025100-c61e-11eb-8ead-4e9a8462b8fa.png">
+
+You've started taking notes, you've started linking ideas, what's next? Time to see your knowledge in action! The graph view is a bird’s eye view of your database, it's a way to identify connections and patterns in your graph. As you continue to take notes, you will start noticing clusters forming which, in addition to its functional purpose, is a way to marvel at your growing database. To access the global graph view, click on the icon in the toolbar or use the global command, `alt/option-g`. 
+
+<img width="258" alt="Screen Shot 2021-06-05 at 11 41 27 AM" src="https://user-images.githubusercontent.com/80150109/120890171-979e8900-c61e-11eb-89b5-6f284299ef0f.png">
+
+## View Customization in the Graph View
+You can customize the graph's forces and choose to filter out the daily notes and orphan nodes. The forces of the graph can be manipulated by changing the link distance and the attraction force of the nodes in the graph under the forces menu in the right corner of the graph view. Under the nodes menu, you can choose to hide daily notes and/or orphan nodes and also change the number of links that are highlighted upon hovering of the individual nodes. 
+
+## The Local Graph
+The local graph is one of the most powerful aspects of the graph view in Athens and allows you to get a catch all view of the pages that are linked to the current  page. It can even let you go a step further by showing you pages that are linked to the linked pages. It's excellent to identify similar topics and pages which you might not have considered as relevant. To open the local graph, click the three dots next to the title of the page to reveal a dropdown menu. In the dropdown menu, click the `Show Local Graph` button.
```

Comment:
```suggestion
The local graph is one of the most powerful aspects of the graph view in Athens and allows you to get a catch all view of the pages that are linked to the current  page. It can even let you go a step further by showing you pages that are linked to the linked pages. It's excellent to identify similar topics and pages which you might not have considered as relevant. 

To open the local graph, click the three dots next to the title of the page to reveal a dropdown menu. In the dropdown menu, click the **Show Local Graph** button.
```